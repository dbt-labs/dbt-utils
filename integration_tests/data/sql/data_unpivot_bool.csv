customer_id,created_at,status,segment,is_updated
123,2017-01-01,active,tier 1,TRUE
234,2017-02-01,active,tier 3,FALSE
567,2017-03-01,churned,tier 2,
