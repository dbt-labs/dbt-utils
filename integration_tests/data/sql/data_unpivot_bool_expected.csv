customer_id,created_at,prop,val
123,2017-01-01,segment,tier 1
123,2017-01-01,status,active
123,2017-01-01,is_updated,true
234,2017-02-01,segment,tier 3
234,2017-02-01,status,active
234,2017-02-01,is_updated,false
567,2017-03-01,status,churned
567,2017-03-01,is_updated,
567,2017-03-01,segment,tier 2
